packages/strapi-database/lib/connector-registry.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+/**
+ * Database connector registry
+ */
+
+const _ = require('lodash');
+const requireConnector = require('./require-connector');
+
+const createConnectorRegistry = ({ defaultConnection, connections }) => {
+  const _connectors = new Map();
+
+  return {
+    /**
+     * Load connector modules
+     */
+    load() {
+      for (const connection of Object.values(connections)) {
+        const { connector } = connection;
+        if (!_connectors.has(connector)) {
+          _connectors.set(connector, requireConnector(connector)(strapi));
+        }
+      }
+    },
+
+    /**
+     * Initialize connectors
+     */
+    async initialize() {
+      for (const connector of _connectors.values()) {
+        await connector.initialize();
+      }
+    },
+
+    get(key) {
+      return _connectors.get(key);
+    },
+
+    set(key) {
+      return _connectors.get(key);
```

Comment:
Is this the desired behavior? / Duplicate of `get`

Reply:
not at all lol bad cope/paste 🤦